service-front/module/Application/view/application/general/home/accessibility.twig

```diff
@@ -47,7 +47,7 @@
     <h2 class="heading-medium">Reporting accessibility problems with this service</h2>
     <p>We’re always looking to improve the accessibility of this service. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, please contact us. </p>
     <h3 class="heading-small">Email</h4>
-    <p><a href="mailto:customerservices@publicguardian.gov.uk" target="_blank">customerservices@<wbr>publicguardian.gov.uk</a></p>
+    <p><a href="mailto:customerservices@publicguardian.gov.uk">customerservices@<wbr>publicguardian.gov.uk</a></p>
```

Comment:
Same as above. Are we 100% its OK to get rid of these target blanks for mails?

Reply:
My reasoning is that this is a mailto: link, and would therefore be expected to open either a new window or a separate application. The fact this is sign-posted in the href suggests we don't need to add a target attribute. Also, if we do have a target attribute, we would need to highlight in the text that this link will open a new tab. It won't necessarily, as the user may have an email application which opens on a click of these links. @johnnolan may have some thoughts.

Reply:
I've discussed with @johnnolan and he concurs that we can do without target="_blank" on mailto links as the browser/screenreader will interpret the mailto href appropriately for the user.